airbyte-config/init/src/main/java/io/airbyte/config/init/CombinedConnectorCatalogDownloader.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.config.init;
+
+import io.airbyte.commons.io.IOs;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.config.CombinedConnectorCatalog;
+import java.nio.file.Path;
+
+public class CombinedConnectorCatalogDownloader {
+
+  public static void main(final String[] args) throws Exception {
+    final Path outputRoot = Path.of(args[0]);
+    final String outputFileName = args[1];
+
+    // TODO (ben) - get url from config
+    final RemoteDefinitionsProvider remoteDefinitionsProvider =
+        new RemoteDefinitionsProvider("https://storage.googleapis.com/prod-airbyte-cloud-connector-metadata-service/oss_catalog.json", 10000);
```

Comment:
TODO: Apply https://github.com/airbytehq/airbyte-platform-internal/pull/4449 so we can use the "better" url

Reply:
Looks like there is a problem with tfstate, if we can't get that sorted we should just merge this API as-is with the understanding that we will bring in the new url shortly thereafter.